src/Encoder.ts

```diff
@@ -13,14 +13,14 @@ export type EncoderOptions<ContextType = undefined> = Partial<
     extensionCodec: ExtensionCodecType<ContextType>;
 
     /**
-     * Encodes bigint as Int64 or Uint64 if it's set to true.
+     * Encodes bigint as Int64 or Uint64 if it's set to true, regardless of the size of bigint number.
      * {@link forceIntegerToFloat} does not affect bigint.
```

Comment:
```suggestion
     * {@link forceIntegerToFloat} does not affect bigint if this is enabled.
```

This is my suggestion to fix the inconsistency. This looks good to merge after this is addresses